deepchem/models/chemberta.py

```diff
@@ -0,0 +1,83 @@
+import torch
+import torch.nn as nn
+from torch.nn import CrossEntropyLoss, MSELoss
+from transformers.modeling_roberta import (
+    ROBERTA_PRETRAINED_MODEL_ARCHIVE_LIST,
+    BertPreTrainedModel,
+    RobertaClassificationHead,
+    RobertaConfig,
+    RobertaModel,
+)
+
+
+class ChemBERTaforSequenceClassification(BertPreTrainedModel):
```

Comment:
I'd suggest that the top level model should be named `ChemBERTa` and should inherit from `dc.models.Model`. If we need a `BertPreTrainedModel` instance, we should maintain it internally, like we do in `KerasMode`l

Reply:
Ah, so in order to get the BERTPreTrainedModel instance to work, we would have to create a wrapper for it that calls BERTPreTrainedModel, or have a copy of the module in dc.models?

Reply:
Would it be better to structure the model class similar to this? It would call Huggingface's RoBERTa sequence classification model for which BertPretTrainedModel is the super class. https://github.com/ThilinaRajapakse/simpletransformers/blob/master/simpletransformers/classification/classification_model.py

Reply:
I'd suggest that the model be renamed to `ChemBERTaSeqClassifier`.  What do you think of longer model names @rbharath ?

Reply:
The way we do it in `KerasModel` is to have a field `self.model` (https://github.com/deepchem/deepchem/blob/master/deepchem/models/keras_model.py#L143) that holds a keras model object that it uses under the hood. I think the idea here is that we would have a `self.model` field that works similarly that holds the `BertPreTrainedModel` instance we need.

CC @peastman who might have some design ideas on how to structure this well

Reply:
Ah I think I get what you're saying now, thanks!

Reply:
Just seeing @vsomnath's comments :). Time lag on refresh for me. I'm tentatively in favor of simpler names, but `ChemBERTaSeqModel` would also be good! (ideally we should have a `mode` argument that allows it to both classify and regress)

Reply:
Agreed! The other models all take a `mode` argument.

Reply:
I haven't tried ChemBERTa on regression tasks yet, but open to adding it once I try experimenting on my local repo!